docs/framework/unmanaged-api/debugging/icordebugmodule3-interface.md

```diff
@@ -52,7 +52,7 @@ interface ICorDebugModule3 : IUnknown
   
  **Library:** CorGuids.lib  
   
- **.NET Framework Versions:**4.5, 4, 3.5 SP1  
+ **.NET Framework Versions:** 4.5, 4, 3.5 SP1
```

Comment:
While you're in here, should "4" be "4.0"?

Reply:
4 is kind of the official name of that release. I'd have to look into other places we use 4 and change it to 4.0 at once.